### PR TITLE
AK-13448 Add Description field for Billing Tag

### PR DESCRIPTION
**Pull request by spikewang**
Add support of Description field for Billing Tag.

### CODE REVIEW

**Comment by spikewang** [comment]
Also, switch the ID to from int to string to be consistent with Terraform concept of Id.

File: alkira/resource_alkira_billing_tag.go [hunk 1]
```diff
@@ -11,17 +10,22 @@ import (
...
 				Required:    true,
 			},
+			"description": {
+				Description: "Billing Tag Description.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
 			"tag_id": {
 				Description: "Billing Tag ID.",
 				Type:        schema.TypeInt,
```
Comment (rvichery) [inline alkira/resource_alkira_billing_tag.go:31]: Should we keep this one as TypeInt? seems like we are only using it as a string

**Review by rvichery — APPROVED** [review]
Looks good to me

File: alkira/resource_alkira_billing_tag.go (hunk 1 above)
Comment (spikewang) [reply alkira/resource_alkira_billing_tag.go:31]: I should remove this one now... No need anymore. The default Id should just work..

**Pull request merged**